app/ui/src/app/customizations/tech-extensions/import/tech-extension-import.component.ts

```diff
@@ -9,10 +10,10 @@ import {
   ParsedResponseHeaders
 } from 'ng2-file-upload';
 import { NotificationType } from 'patternfly-ng';
-import { Extension } from '../../../model';
-import { ExtensionStore } from '../../../store/extension/extension.store';
-import { NotificationService } from 'app/common/ui-patternfly/notification-service';
-import {Observable} from "rxjs/Observable";
+
+import { Extension } from '@syndesis/ui/model';
```

Comment:
👍

Reply:
This was unrelated to this issue, but as I pointed out earlier, any software engineer that worths this title should observe the Boy Scout Rule with no exception, and I found this when linting the PR.

Reply:
I don't believe in separate PRs for cleanups like this, or they'll never get done. 👍